app/commands/v2/put_content.rb

```diff
@@ -44,9 +44,11 @@ def update_content_dependencies(edition)
       end
 
       def create_links(edition)
-        edition.copy_links_from(previously_published_item.links)
+        if payload[:links].nil?
```

Comment:
I think it'd be more readable if this had an `if else` to make it clearer that copying and creating are exclusive